x-pack/plugins/security_solution/public/management/pages/policy/models/advanced_policy_schema.ts

```diff
@@ -15,301 +17,281 @@ export const AdvancedPolicySchema: AdvancedPolicySchemaType[] = [
   {
     key: 'linux.advanced.agent.connection_delay',
     first_supported_version: '7.11',
-    documentation: '',
+    documentation: i18n.translate(
+      'xpack.securitySolution.endpoint.policy.advanced.linux.advanced.agent.connection_delay',
+      {
+        defaultMessage:
+          'How long to wait for agent connectivity before sending first policy reply, in seconds. Default: 60.',
+      }
+    ),
   },
   {
     key: 'linux.advanced.artifacts.global.base_url',
     first_supported_version: '7.11',
-    documentation: '',
+    documentation:
+      'Base URL from which to download global artifact manifests. Default: https://artifacts.security.elastic.co.',
   },
   {
     key: 'linux.advanced.artifacts.global.manifest_relative_url',
     first_supported_version: '7.11',
-    documentation: '',
-  },
-  {
-    key: 'linux.advanced.artifacts.global.ca_cert',
-    first_supported_version: '7.11',
-    documentation: '',
+    documentation:
+      'Relative URL from which to download global artifact manifests. Default: /downloads/endpoint/manifest/artifacts-<version>.zip.',
   },
   {
     key: 'linux.advanced.artifacts.global.public_key',
     first_supported_version: '7.11',
-    documentation: '',
+    documentation: 'Public key used to verify the global artifact manifest signature.',
   },
   {
     key: 'linux.advanced.artifacts.global.interval',
     first_supported_version: '7.11',
-    documentation: '',
-  },
-  {
-    key: 'linux.advanced.artifacts.user.base_url',
-    first_supported_version: '7.11',
-    documentation: '',
-  },
-  {
-    key: 'linux.advanced.artifacts.user.ca_cert',
-    first_supported_version: '7.11',
-    documentation: '',
+    documentation:
+      'Interval between global artifact manifest download attempts, in seconds. Default: 3600.',
   },
   {
     key: 'linux.advanced.artifacts.user.public_key',
     first_supported_version: '7.11',
-    documentation: '',
+    documentation: 'Public key used to verify the user artifact manifest signature.',
   },
   {
     key: 'linux.advanced.artifacts.user.interval',
     first_supported_version: '7.11',
-    documentation: '',
+    documentation:
+      'Interval between user artifact manifest download attempts, in seconds. Default: 3600.',
```

Comment:
@brian-mckinney does this really exist? Or are user artifacts only downloaded on-demand when policy is applied?

Reply:
user artifacts are only downloaded on demand when policy is applied.  Any interval specified applies to global artifacts only

Reply:
ok, removing all `user.interval` fields